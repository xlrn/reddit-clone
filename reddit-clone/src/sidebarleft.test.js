import renderer from 'react-test-renderer';
import SideBarLeft from './sidebarleft';

it ('renders properly', () => {
    const component = renderer.create(
        <SideBarLeft/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})