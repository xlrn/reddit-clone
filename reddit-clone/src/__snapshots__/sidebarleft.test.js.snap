// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="col-3"
>
  placeholder
</div>
`;
